hibernate-core/src/main/java/org/hibernate/dialect/CockroachDB1920Dialect.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect;
+
+import org.hibernate.dialect.identity.CockroachDB1920IdentityColumnSupport;
+import org.hibernate.dialect.identity.IdentityColumnSupport;
+import org.hibernate.hql.spi.id.MultiTableBulkIdStrategy;
+import org.hibernate.hql.spi.id.inline.InlineIdsInClauseBulkIdStrategy;
+import org.hibernate.type.descriptor.sql.*;
+
+import java.sql.*;
+
+import java.sql.DatabaseMetaData;
+
+/**
+ * An SQL dialect for CockroachDB 19.2 and later. This is the first dialect for CockroachDB. It extends
+ * {@link PostgreSQL95Dialect} because CockroachDB is aiming for Postgres compatibility.
+ */
+public class CockroachDB1920Dialect extends PostgreSQL95Dialect {
+
+    public CockroachDB1920Dialect() {
+        super();
+        registerColumnType( Types.BLOB, "bytea" );
+    }
+
+    @Override
+    public IdentityColumnSupport getIdentityColumnSupport() {
+        return new CockroachDB1920IdentityColumnSupport();
+    }
+
+    @Override
+    public MultiTableBulkIdStrategy getDefaultMultiTableBulkIdStrategy() {
+        // CockroachDB 19.2.0 does not support temporary tables, so we must override the Postgres behavior.
```

Comment:
if we aim not only for one single version, we'd better change the verbiage to something like 'CockroachDB 19.2.0 or later ...'.